ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceTaskFragmentTest.kt

```diff
@@ -149,6 +154,29 @@ class MultipleChoiceTaskFragmentTest :
     buttonIsEnabled("Skip")
   }
 
+  @Test
+  fun testActionButtons_dataEntered_skipButtonTapped_confirmationDialogIsShown() {
+    val multipleChoice = MultipleChoice(options, MultipleChoice.Cardinality.SELECT_ONE)
+    setupTaskFragment<MultipleChoiceTaskFragment>(task.copy(multipleChoice = multipleChoice))
+
+    onView(withText("Option 1")).perform(click())
+
+    onView(withText("Skip")).perform(click())
+    onView(withText(context.getString(R.string.skip_dialog_title)))
+      .inRoot(isDialog())
+      .check(matches(isDisplayed()))
```

Comment:
Robolectric has a utility [`ShadowDialog`](https://robolectric.org/javadoc/4.3/org/robolectric/shadows/ShadowDialog.html) that can be used here. 

e.g. ShadowDialog.getLatestDialog(), ShadowDialog.getShownDialogs()